docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,254 @@
+
+You can access emails in Mailpit's Web UI at [http://localhost:8075](http://localhost:8075).
+
+### Desktop Modeler
+
+:::info
+Desktop Modeler is [open source and free to use](https://github.com/camunda/camunda-modeler).
 :::
 
-This Docker Compose configuration serves two purposes:
+[Download the Desktop Modeler](https://camunda.com/download/modeler/) and start modeling BPMN, DMN and Camunda Forms on your local machine.
+
+### Use the CLI
+
+<Tabs groupId="cli" defaultValue="with" queryString values={
+[
+{label: 'With Zeebe request authentication', value: 'with' },
+{label: 'Without authentication', value: 'without' },
+]}>
+
+<TabItem value="with">
+
+If you enabled authentication for GRPC requests on Zeebe, provide the following client credentials when deploying and executing processes:
+
+- Authentication: `OAuth`
+- URL: `http://localhost:26500`
+- Client ID: `zeebe`
+- Client secret: `zecret`
+- OAuth URL: `http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token`
+- Audience: `zeebe-api`
+
+</TabItem>
+
+<TabItem value="without">
+
+Once you are ready to deploy or execute processes, use the following settings to deploy to the local Zeebe instance:
+
+- Authentication: `None`
+- URL: `http://localhost:26500`
+
+</TabItem>
+</Tabs>
+
+## Secure the Zeebe API
 
-1. It can be used to start up a development environment locally.
-2. It documents how the various components need to be wired together.
+By default, the Zeebe GRPC API is publicly accessible without requiring any client credentials for development purposes.
+
+To enable authentication of GRPC requests in Zeebe, setting the environment variable `ZEEBE_AUTHENTICATION_MODE` to `identity`, for example:
+
+```shell
+ZEEBE_AUTHENTICATION_MODE=identity docker compose up -d
+```
+
+The default value can also be modified directly in the included `.env` file.
+
+## Connectors
+
+Both the full and lightweight Docker Compose files contain a configuration for [Out-of-the-box Connectors](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md). Refer to the [Connector installation guide](/self-managed/connectors-deployment/install-and-start.md) for details on how to provide the related Connector templates for modeling.
+
+### Connector secrets
+
+Secrets can be added into the Connector runtime using the included `connector-secrets.txt` file. Add secrets in the format `NAME=VALUE`
+per line. The secrets will then be available in the Connector runtime in the format `secrets.NAME`.
+
+### Custom connectors
+
+To add custom Connectors, create a new Docker Image bundling them as described in the [Connectors repository](https://github.com/camunda/connectors).
+
+Alternatively, you can mount new Connector JARs as volumes into the `/opt/app` folder by adding this to the Docker Compose file. Keep in mind that the Connector JARs must include all necessary dependencies inside the JAR.
+
+## Kibana
```

Comment:
I've formatted/edited these instructions, but the Kibana profile doesn't run for me in the provided 8.6 or alpha/8.7 compose file. Not sure if this is an error in my setup or in the profile. Can you verify Kibana works and that these instructions are correct?

Reply:
I've tried it with Camunda 8.6 Docker Compose, and it works as expected.
What issue/error do you have?